scripts/build/builders/tizen.py

```diff
@@ -191,11 +192,29 @@ def GnBuildArgs(self):
                 raise Exception(
                     "Environment %s missing, cannot build Tizen target" % env)
 
+        sysroot = None
+
+        if self.board.value.target_cpu == "arm64":
+            env = 'TIZEN_SDK_SYSROOT_ARM64'
+            if env not in os.environ:
+                raise Exception(
+                    "Environment %s missing, cannot build Tizen target" % env)
+
+            sysroot = os.environ[env]
+
+        elif self.board.value.target_cpu == "arm":
+            env = 'TIZEN_SDK_SYSROOT'
```

Comment:
Please check that everything works fine if `TIZEN_SDK_SYSROOT` is not defined: `unset TIZEN_SDK_SYSROOT`